tests/dummy/app/styles/app.scss

```diff
@@ -18,7 +17,6 @@
     white-space: -o-pre-wrap;
   }
   code {
-    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
```

Comment:
I was thinking we were removing the `font-family` settings in the components themselves, not the demos. Then again, we should probably have `ember-frost-css-theme` or whatever provide the right styles for code samples... so I guess this is fine.